core/controllers/collection_editor.py

```diff
@@ -155,6 +155,44 @@ def _require_valid_version(self, version_from_payload, collection_version):
                 % (collection_version, version_from_payload))
 
     @require_editor
+    def get(self, collection_id):
+        """Populates the data on the individual collection page."""
+
+        collection = (
+            collection_services.get_collection_by_id(collection_id))
+
+        if (collection is None or
```

Comment:
Is this check needed when we use ``@require_editor``? I don't think so because that requires the user to have edit access to the collection, whereas we're only checking view access here. Users with edit access automatically have view access.